test/integration/targets/hwc_evs_disk/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: delete a disk
+  hwc_evs_disk:
+    filters:
```

Comment:
The hwc_evs_disk module support those following values to filter:
"availability_zone,name,volume_type,backup_id,description,enable_full_clone,enable_scsi,enable_share,encryption_id,image_id,size,snapshot_id". The "name" is not only one value to filter. 

How should a user decide which one to use?
---It's up to what kind of disk the user need. If they want 10G size and scsi type disk, then they should add "size=10 and enable_scsi=true" values in filter.

Many modules use name as the a preset identifier and fail in case 2 identical names where found (because they usually not always unique).
--Then we cloud add other values in filters

Reply:
The hwc_evs_disk module support those following values to filter:
"availability_zone,name,volume_type,backup_id,description,enable_full_clone,enable_scsi,enable_share,encryption_id,image_id,size,snapshot_id". The "name" is not only one value to filter.

How should a user decide which one to use?
---It's up to what kind of disk the user need. If they want 10G size and scsi type disk, then they should add "size=10 and enable_scsi=true" values in filter.

Many modules use name as the a preset identifier and fail in case 2 identical names where found (because they usually not always unique).
--Then we cloud add other values in filters

Reply:
> It's up to what kind of disk the user need. If they want 10G size and scsi type disk, then they should add "size=10 and enable_scsi=true" values in filter.

If I am get that right, a user woud have to use it this way?

```
- name: create a disk
  hwc_evs_disk:
    filters:
      - name
      - size=10
      - enable_scsi=true
    availability_zone: "cn-north-1a"
    name: "ansible_evs_disk_test"
    volume_type: "SATA"
    size: 10
```
I am sorry to say but this is not how ansible is meant to work: a user already gave you this info in the params, right?
```
- name: create a disk
  hwc_evs_disk:
    availability_zone: "cn-north-1a"
    name: "ansible_evs_disk_test"
    volume_type: "SATA"
    size: 10
```
you as a module developer must decide whether these params are enough to uniquely identify the resource and if not (if it exists).